inst/extdata/aglu/USDA_crops.csv

```diff
@@ -0,0 +1,16 @@
+# File: USDA_crops.csv,,
```

Comment:
Here and elsewhere, please remove ugly Excel trailing commas at end of header lines